tools/releaseBuild/azureDevOps/templates/linux.yml

```diff
@@ -148,6 +160,8 @@ jobs:
       value: false
     - name: NugetSecurityAnalysisWarningLevel
       value: none
+    - name: skipComponentGovernanceDetection
```

Comment:
I don't see where this variable is used...

Reply:
it is used by Azure DevOps